libpromises/verify_reports.c

```diff
@@ -147,31 +147,57 @@ static void ReportToFile(const char *logfile, const char *message)
     }
 }
 
-static bool PrintFile(const char *filename, size_t max_lines)
+static bool PrintFile(const char *filename, int max_lines)
 {
     if (!filename)
     {
-        Log(LOG_LEVEL_VERBOSE, "Printfile promise was incomplete, with no filename.");
+        Log(LOG_LEVEL_VERBOSE,
+            "Printfile promise was incomplete, with no filename.");
```

Comment:
Please don't make changes like this or put them into a separate commit.

Reply:
Moved code foldings to a separate commit